src/nnet3/nnet-compute.h

```diff
@@ -124,6 +157,29 @@ class NnetComputer {
   void GetOutputDestructive(const std::string &output_name,
                             CuMatrix<BaseFloat> *output);
 
+  // Get the matrices where store the state for next computation
+  inline const std::vector< CuMatrix<BaseFloat> > &GetMatrices() const { 
+    return matrices_; }
+
+  // Get state of streams. The parameter named "batch_first" indicates whether 
+  // the matrices saved state is batch first or not.
```

Comment:
Need more documentation of this interface: what it does, and what the args mean; for vectors, what their sizes are.  What is batch_size?  Must it be the same as the batch size of the NnetComputation, or just <= that?

Reply:
.. same for SetState().